anyblok_pyramid/tests/test_adapter.py

```diff
@@ -39,6 +42,59 @@ def add_route_and_views(config):
         assert res.json_body['datetime'] == datetime_adapter(
             datetime(2017, 10, 1, 1, 1, 1), None)
 
+    def test_registry_get_timedelta(self):
+
+        def get_timedelta(request):
+            return {'timedelta': timedelta(
+                days=1, hours=4, minutes=56,
+                seconds=3710, milliseconds=4000,
+                microseconds=500)}
+
+        def add_route_and_views(config, mode='seconds'):
+            config.add_route('dbname', '/test/')
+            config.add_view(
+                get_timedelta, route_name='dbname', renderer='json')
+            json_renderer = JSON()
+            json_renderer.add_adapter(
+                timedelta, lambda obj, request: timedelta_adapter(
+                    obj, request, mode))
+            config.add_renderer('json', json_renderer)
```

Comment:
it's complicated to understand the use with a lamda.

Maybe the better action is to decorate the timedelta_adapter

```
def timedelta_factory(mode=SECONDS):
    def timedelta_adapter(obj, request):
          # copy your adapter here

    return wrap
```

so the configuration is easer

```
from anyblok_pyramid.adapter import timedelta_factory, MILLISECONDS

...

json_renderer.add_adapter(timedelta, timedelta_factory(MILLISECONDS))
```